client/my-sites/plans/p2-plans-main.jsx

```diff
@@ -22,6 +23,13 @@ export class P2PlansMain extends Component {
 					hidePlanTypeSelector={ true }
 					intent="plans-p2"
 					isInSignup={ false }
+					showPlanTypeSelectorDropdown={
```

Comment:
`Non-blocking`:

Can address in a follow-up, but I don't believe that the p2 plans page shows a plan type selector